runner/cf_helpers.go

```diff
@@ -18,7 +18,18 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var insecureTransport *http.Transport
+var clientWithInsecureTransport *http.Client
```

Comment:
any reason for this to be global? 🤔

Reply:
@jimbo459 replied:
> As per go doc http.Client:
```
The Client’s Transport typically has internal state (cached TCP
connections), so Clients should be reused instead of created as needed.
Clients are safe for concurrent use by multiple goroutines.```

Reply:
Thanks @kirederik - the thinking here was to implement a singleton pattern and re-use the client, rather than create one every-time the function is called.